bin/rename-wp-commit.sh

```diff
@@ -0,0 +1,16 @@
+#/usr/bin/env bash
+
+# First parameter is the ticket number
+
+# To save the changes
+git commit
+
+# Get the data to print
+commit=`git log wp/master --grep=$1 --oneline --pretty=format:'%h' -n 1`
```

Comment:
Same as for the other script, we need to use a changeset number instead of a ticket number, and `--grep=$1` is not a good way to do the matching.

Reply:
what do you suggest instead grep?